packages/content/src/components/content-header/content-header.tsx

```diff
@@ -0,0 +1,94 @@
+import { useCourseGroupBySlugQuery } from '../../graphql/queries';
+
+const ContentHeader = (props: {
+  contentKind: ContentKind;
+  slug: string;
+  showStars: boolean;
+  showImage: boolean;
+}): JSX.Element => {
+  const StarIcon = (
+    <svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M7.85879 1.51237C8.21879 0.407172 9.78239 0.407172 10.1412 1.51237L11.4252 5.46277C11.5036 5.70336 11.6561 5.91299 11.8609 6.0617C12.0656 6.21041 12.3121 6.29059 12.5652 6.29077H16.7196C17.8824 6.29077 18.3648 7.77877 17.4252 8.46277L14.0652 10.9036C13.86 11.0524 13.7073 11.2624 13.6288 11.5034C13.5503 11.7444 13.5502 12.0041 13.6284 12.2452L14.9124 16.1956C15.2724 17.3008 14.0064 18.2212 13.0644 17.5372L9.70439 15.0964C9.49944 14.9476 9.25266 14.8674 8.99939 14.8674C8.74612 14.8674 8.49935 14.9476 8.29439 15.0964L4.93439 17.5372C3.99359 18.2212 2.72879 17.3008 3.08759 16.1956L4.37159 12.2452C4.4498 12.0041 4.44966 11.7444 4.37119 11.5034C4.29272 11.2624 4.13996 11.0524 3.93479 10.9036L0.575993 8.46397C-0.363607 7.77997 0.119993 6.29197 1.28159 6.29197H5.43479C5.68805 6.29204 5.93484 6.21198 6.13982 6.06325C6.34481 5.91452 6.49748 5.70475 6.57599 5.46397L7.85999 1.51357L7.85879 1.51237Z"
+        fill="#FAAD4D"
+      />
+    </svg>
+  );
+
+  const StarIconShaded = (
+    <svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M7.85879 1.51237C8.21879 0.407172 9.78239 0.407172 10.1412 1.51237L11.4252 5.46277C11.5036 5.70336 11.6561 5.91299 11.8609 6.0617C12.0656 6.21041 12.3121 6.29059 12.5652 6.29077H16.7196C17.8824 6.29077 18.3648 7.77877 17.4252 8.46277L14.0652 10.9036C13.86 11.0524 13.7073 11.2624 13.6288 11.5034C13.5503 11.7444 13.5502 12.0041 13.6284 12.2452L14.9124 16.1956C15.2724 17.3008 14.0064 18.2212 13.0644 17.5372L9.70439 15.0964C9.49944 14.9476 9.25266 14.8674 8.99939 14.8674C8.74612 14.8674 8.49935 14.9476 8.29439 15.0964L4.93439 17.5372C3.99359 18.2212 2.72879 17.3008 3.08759 16.1956L4.37159 12.2452C4.4498 12.0041 4.44966 11.7444 4.37119 11.5034C4.29272 11.2624 4.13996 11.0524 3.93479 10.9036L0.575993 8.46397C-0.363607 7.77997 0.119993 6.29197 1.28159 6.29197H5.43479C5.68805 6.29204 5.93484 6.21198 6.13982 6.06325C6.34481 5.91452 6.49748 5.70475 6.57599 5.46397L7.85999 1.51357L7.85879 1.51237Z"
+        fill="#D1D5DB"
+      />
+    </svg>
+  );
+
+  const StarCount = (props: { rating: number }) => {
+    const ratingArray = [];
+    for (let i = 0; i < 5; i++) {
+      if (i < props.rating / 2) {
+        ratingArray.push(
+          <div key={i} className="px-1">
+            {StarIcon}
+          </div>
+        );
+      } else {
+        ratingArray.push(
+          <div key={i} className="px-1">
+            {StarIconShaded}
+          </div>
+        );
+      }
+    }
+    return <>{ratingArray}</>;
+  };
+
+  const CalculateRating = (rating: number) => {
```

Comment:
I think we can refactor the SVG elements and the math as it feels a bit disjointed at the moment. Some math in a dedicated function, some in the `StarCount` component, then some in the returned component.

It also leads to some 'inaccuracies' in the information provided. Like in the storybook component, with a rating of 36, displayed in a factor of 5, I would expect to see `1.8`. But because of the `Math.round` and then `/ 2) + '.0'` on Line 78, I see an average of `2.0` instead.

Think a cleaner approach would to have a dedicated score for returning the score as a score out of 5, (e.g., calling `calculateRating(36)` would return `1.8`;

Then in the returned component you can just display that value without the additional ` / 2) + '.0'` transformation.

For the stars piece, think we can borrow some inspiration from SO: https://stackoverflow.com/a/71563209

With the `1.8` returned from our `calculateRating` function, we can move the decimal, and pass `18%` to the `<rect>` `width` in our svg, giving us a more accurate percentage of stars filled.

Last thing -- would rename this `calculateRating`. Know it's nitpicky, but we're using camelCase for functions/variables, and PascalCase for components/Classes.

Reply:
Refactor pushed. Some bug changes so let me know if anything needs more attention!

Reply:
@AlexONeillTI So, with the `symbol`/`mask` used in the SVG, we don't actually need to iterate through a star count in the `RenderStars` function or bother with the `full`/`half` `clipId` stuff. We can just use the `mask` to display 5 of the `#star` elements, then set the `width` of our `rect` to `props.score` to achieve the same effect. 

Mocked up an example here if you'd like to implement it: https://github.com/thoughtindustries/helium/compare/CLM-8307-ContentHeader-Component...CLM-8307-ContentHeader-Component-rs?expand=1 Hit me up if anything doesn't make sense or you've got questions on anything.